src/common/telemetry/src/logging.rs

```diff
@@ -96,49 +98,70 @@ pub enum OtlpExportProtocol {
 #[derive(Clone, Debug, Serialize, Deserialize, PartialEq)]
 pub struct SlowQueryOptions {
```

Comment:
Why not derive `serde(default)` for the struct so we don't have to set default for each field?
```suggestion
#[derive(Clone, Debug, Serialize, Deserialize, PartialEq)]
#[serde(default)]
pub struct SlowQueryOptions {
```

Reply:
Good catch! I forgot this feature.

Reply:
Addressed https://github.com/GreptimeTeam/greptimedb/pull/6734/commits/9890173841f2a6d988f37d254a93e54cafb61bc8